src/Symfony/Bridge/PhpUnit/bin/simple-phpunit.php

```diff
@@ -100,8 +102,9 @@
 
 
 $SYMFONY_PHPUNIT_REMOVE = $getEnvVar('SYMFONY_PHPUNIT_REMOVE', 'phpspec/prophecy'.($PHPUNIT_VERSION < 6.0 ? ' symfony/yaml': ''));
+$configurationHash = implode(PHP_EOL, array(md5_file(__FILE__), $SYMFONY_PHPUNIT_REMOVE, (int) $PHPUNIT_REMOVE_RETURN_TYPEHINT));
```

Comment:
hum, what you proposed on Slack here: we should make the path vary instead of the hash, so that caching works

Reply:
done. suffix folder with md5 of configuration